src/store/interfaces.ts

```diff
@@ -35,6 +35,18 @@ export interface IOrdinoAppState {
    * Id of the current focus view
    */
   focusWorkbenchIndex: number;
+
+  /**
+   * Stores the available global query that will be applied for the current session
+   * analog to IOrdinoGlobalQuery
+   */
+  globalQuery?: { id: string; name: string; filter: { col: string; op: string; val: (string | number)[] } };
+
+  /**
+   * Filter query that is applied to all entities.
+   * analog to IQueryFilter
+   */
+  appliedQueryFilter?: { col: string; op: string; val: (string | number)[] };
```

Comment:
```suggestion
  appliedQueryFilter?: IQueryFilter;
```

Can we use the interface here instead?